js/mainApiMgr.js

```diff
@@ -98,6 +99,10 @@ electron.ipcMain.on(apiName, (event, arg) => {
         // renderer window that has a registered activity detection from JS.
         activityDetection.setActivityWindow(arg.period, event.sender);
     }
+
+    if (arg.cmd === apiCmds.showAlertSettings) {
+        configureAlert.openConfigurationWindow();
```

Comment:
Can you rename this to "openNotificationConfiguration" or "openAlertConfiguration"?